mm/shm/shmdt.c

```diff
@@ -33,32 +33,34 @@
 #include <nuttx/sched.h>
 #include <nuttx/mm/shm.h>
 #include <nuttx/pgalloc.h>
+#include <nuttx/mm/map.h>
 
 #include "shm/shm.h"
 
 #ifdef CONFIG_MM_SHM
 
 /****************************************************************************
- * Public Functions
+ * Private Functions
```

Comment:
shmdt_priv isn't static function

Reply:
done, moved to Public. I considered it "private" because it is not supposed to be called outside this driver